WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js

```diff
@@ -2533,6 +2533,11 @@ ZaAccount.isEmailRetentionPolicyEnabled = function () {
     try {
         var instance  = this.getInstance () ;
         var gc   = ZaApp.getInstance().getGlobalConfig();
+        if (!instance.attrs[ZaAccount.A_mailHost]) {
```

Comment:
safer check would be !AjxUtil.isEmpty(this.attrs[ZaAccount.A_mailHost])

Reply:
modified to AjxUtil.isEmpty(instance.attrs[ZaAccount.A_mailHost])